lib/data/item.dart

```diff
@@ -2,37 +2,67 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 
 // In the future we will want to override this class to lost vs. found objects
 class Item {
-  final String? itemName;
-  final String? description;
-  final num? phone;
-  final String? picture;
-  final GeoPoint? location;
+  final String id;
```

Comment:
What is id for?

Reply:
Pretty sure we don't use it right now, but id is the document ID of the item on our backend. If we plan to be able to modify a listing or delete a post, then this would be necessary.